### PR TITLE
Add :-moz-system-metric(…)

### PR DESCRIPTION
**Pull request by ExE-Boss**
This adds the browser compatibility table data for the <code>:-moz-system-metric([scrollbar-end-backward](https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-system-metric(scrollbar-end-backward)), [scrollbar-end-forward](https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-system-metric(scrollbar-end-forward)), [scrollbar-start-backward](https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-system-metric(scrollbar-start-backward)), [scrollbar-start-forward](https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-system-metric(scrollbar-start-forward)), [scrollbar-thumb-proportional](https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-system-metric(scrollbar-thumb-proportional)), [touch-enabled](https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-system-metric(touch-enabled)) and [windows-default-theme](https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-system-metric(windows-default-theme)))</code> CSS selector.

### CODE REVIEW

File: css/selectors/-moz-system-metric.json [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+            },
+            "uc_android": {
+              "version_added": false
+            },
+            "uc_chinese_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not experimental, purely internal. Set it to false.

File: css/selectors/-moz-system-metric.json [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+              },
+              "uc_android": {
+                "version_added": false
+              },
+              "uc_chinese_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not experimental, purely internal. Set it to false.

File: css/selectors/-moz-system-metric.json [hunk 3]
```diff
@@ -0,0 +1,525 @@
...
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not experimental, purely internal. Set it to false.

File: css/selectors/-moz-system-metric.json [hunk 4]
```diff
@@ -0,0 +1,525 @@
...
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not experimental, purely internal. Set it to false.

File: css/selectors/-moz-system-metric.json [hunk 5]
```diff
@@ -0,0 +1,525 @@
...
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not experimental, purely internal. Set it to false.

File: css/selectors/-moz-system-metric.json [hunk 6]
```diff
@@ -0,0 +1,525 @@
...
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not experimental, purely internal. Set it to false.

File: css/selectors/-moz-system-metric.json [hunk 7]
```diff
@@ -0,0 +1,525 @@
...
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not experimental, purely internal. Set it to false.

File: css/selectors/-moz-system-metric.json [hunk 8]
```diff
@@ -0,0 +1,525 @@
...
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not experimental, purely internal. Set it to false.

File: css/selectors/-moz-system-metric.json [hunk 9]
```diff
@@ -0,0 +1,525 @@
...
+            "uc_android": {
+              "version_added": false
+            },
+            "uc_chinese_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not a standard track.

File: css/selectors/-moz-system-metric.json [hunk 10]
```diff
@@ -0,0 +1,525 @@
...
+            "deprecated": false
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Deprecated on the Web: true

File: css/selectors/-moz-system-metric.json [hunk 11]
```diff
@@ -0,0 +1,525 @@
...
+              "uc_android": {
+                "version_added": false
+              },
+              "uc_chinese_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not on a standard track

File: css/selectors/-moz-system-metric.json [hunk 12]
```diff
@@ -0,0 +1,525 @@
...
+              "deprecated": false
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Deprecated on the web

File: css/selectors/-moz-system-metric.json [hunk 13]
```diff
@@ -0,0 +1,525 @@
...
+              "uc_android": {
+                "version_added": false
+              },
+              "uc_chinese_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not a standard track.

File: css/selectors/-moz-system-metric.json [hunk 14]
```diff
@@ -0,0 +1,525 @@
...
+              "deprecated": false
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Deprecated on the web

File: css/selectors/-moz-system-metric.json [hunk 15]
```diff
@@ -0,0 +1,525 @@
...
+              "uc_android": {
+                "version_added": false
+              },
+              "uc_chinese_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not a standard track.

File: css/selectors/-moz-system-metric.json [hunk 16]
```diff
@@ -0,0 +1,525 @@
...
+              "deprecated": false
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Deprecated on the web

File: css/selectors/-moz-system-metric.json [hunk 17]
```diff
@@ -0,0 +1,525 @@
...
+              "uc_android": {
+                "version_added": false
+              },
+              "uc_chinese_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not a standard track.

File: css/selectors/-moz-system-metric.json [hunk 18]
```diff
@@ -0,0 +1,525 @@
...
+              "deprecated": false
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Deprecated on the web.

File: css/selectors/-moz-system-metric.json [hunk 19]
```diff
@@ -0,0 +1,525 @@
...
+              "uc_android": {
+                "version_added": false
+              },
+              "uc_chinese_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not a standard track.

File: css/selectors/-moz-system-metric.json [hunk 20]
```diff
@@ -0,0 +1,525 @@
...
+              "deprecated": false
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Deprecated on the web.

File: css/selectors/-moz-system-metric.json [hunk 21]
```diff
@@ -0,0 +1,525 @@
...
+              "uc_android": {
+                "version_added": false
+              },
+              "uc_chinese_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not a standard track.

File: css/selectors/-moz-system-metric.json [hunk 22]
```diff
@@ -0,0 +1,525 @@
...
+              "deprecated": false
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Deprecated on the web

File: css/selectors/-moz-system-metric.json [hunk 23]
```diff
@@ -0,0 +1,525 @@
...
+              "uc_android": {
+                "version_added": false
+              },
+              "uc_chinese_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Not a standard track

File: css/selectors/-moz-system-metric.json [hunk 24]
```diff
@@ -0,0 +1,525 @@
...
+              "deprecated": false
```
Comment (teoli2003) [inline css/selectors/-moz-system-metric.json]: Deprecated on the web.

**Comment by ExE-Boss** [comment]
I accidentally missed that because I copied the data from a template that I use and forgot to update the `status`. I’ve corrected it now.

**Pull request merged**